packages/arb-token-bridge-ui/src/components/TransferPanel/OneNovaTransferDialog.tsx

```diff
@@ -0,0 +1,94 @@
+import { Dialog as HeadlessUIDialog, Tab } from '@headlessui/react'
+import { XMarkIcon } from '@heroicons/react/24/outline'
+import Hop from '@/images/bridge/hop.png'
+
+import { useAppState } from '../../state'
+import { TabButton } from '../common/Tab'
+import { BridgesTable } from '../common/BridgesTable'
+import { Dialog, UseDialogProps } from '../common/Dialog'
+import { FastBridgeInfo, FastBridgeNames } from '../../util/fastBridges'
+import { ChainId, getNetworkName } from '../../util/networks'
+
+export function OneNovaTransferDialog(
+  props: UseDialogProps & {
+    destinationChainId: number | null
+    amount: string
+  }
+) {
+  const {
+    app: { selectedToken }
+  } = useAppState()
+
+  const { destinationChainId } = props
+
+  if (!destinationChainId) {
+    return null
+  }
+
+  const sourceChainId =
+    destinationChainId === ChainId.ArbitrumNova
+      ? ChainId.ArbitrumOne
+      : ChainId.ArbitrumNova
+
+  const sourceNetworkSlug =
+    sourceChainId === ChainId.ArbitrumOne ? 'arbitrum' : 'nova'
+  const destinationNetworkSlug =
+    destinationChainId === ChainId.ArbitrumOne ? 'arbitrum' : 'nova'
+
+  const bridgeDeepLink = `https://app.hop.exchange/#/send?sourceNetwork=${sourceNetworkSlug}&destNetwork=${destinationNetworkSlug}&token=${
+    selectedToken?.symbol || 'ETH'
+  }&amount=${props.amount}`
+
+  // only enable Hop for now
+  const fastBridgeList: FastBridgeInfo[] = [
+    { name: FastBridgeNames.Hop, imageSrc: Hop, href: bridgeDeepLink }
```

Comment:
We don't pull fast bridges with `getFastBridges` because the method would have to be refactored to allow getting specific bridge data. Right now we have to pass multiple props and we get fast bridges based on network IDs, selected tokens, etc. As seen [here](https://github.com/OffchainLabs/arbitrum-token-bridge/blob/master/packages/arb-token-bridge-ui/src/util/fastBridges.ts#L49)

Reply:
Fair, the `getFastBridges` is ripe for refactoring anyway